CrisisTextLine/Elmer
# Platform rewrite

#### What's this PR do?
rewrites the platform script to handle the new automated graduating process
#### Where should the reviewer start?
jobs/training/pollCanvasForGraduatedUsersThenCreatePlatformAccount.js
#### How should this be manually tested?
create a test user, publish Trainer's Review assignment in a course, give the test user passing grades for everything EXCEPT platform ready assignment. run the script, it should ONLY graduate that test user.
#### Any background context you want to provide?
#### What are the relevant tickets?
https://admin.crisistextline.org/jira/browse/INT-294
#### Questions:
